### DISCUSSION TITLE
Annotation without the 'line' parameter should not default to line 1

### DISCUSSION THREAD

**Discussion by pdostal** [community/community#38154] [Actions]
Hello, if I have annotation without `line` parameter, that annotation looks like this:
![](https://user-images.githubusercontent.com/1254493/198281872-366fb305-8d33-4f74-b683-faee96c5047a.png)

It's fine to see this annotation at the beginning of selected file but following line is wrong:
> Check failure on line 1 in tests/console/sshd.pm

The line is wrong because [the documentation](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-system-path) allows the `line` parameter being optimal.

I would appreciate to just not mention the first line when the `line` parameter is not provided. Does it make sense to you?

**Comment by ossanna16** [comment]
Hi there @pdostal and welcome to our community! Thank you for the feedback 🙂

**Reply by pdostal** [reply]
Hello, is there a chance that this report will get some attention from GitHub developers?

**Reply by ossanna16** [reply]
Hi @pdostal! Yes, our product teams keep a close eye on this space 👍

**Reply by foursixnine** [reply]
Hi @ossanna16 any updates on this?
